test/spec/ol/render/canvas/index.test.js

```diff
@@ -1,10 +1,72 @@
-
-
+goog.require('ol.events');
+goog.require('ol.obj');
 goog.require('ol.render.canvas');
 
 
 describe('ol.render.canvas', function() {
 
+  var font = document.createElement('link');
+  font.href = 'https://fonts.googleapis.com/css?family=Inconsolata';
```

Comment:
this will make it impossible to run our tests offline but I assume that is already the case (haven't checked)?

Reply:
Yes, our tests already require an internet connection.